CGPs/cgp-0107.md

```diff
@@ -0,0 +1,169 @@
+||Guide and train traditional developers in-person on how to build on Celo…|Venue: $1,500 - $500 per day for 3 days.||
+||Focus on building a climate neutral initiative with Celo to tackle climate change.|Refreshment - $3,500 for 3 days||
+||Focus on integrating Celo wallets to provide payment solution to businesses|Promotion: $1,000||
+||Build applications on Celo that will enhance user onboarding experience to web 3.0.|Prize for winners: $5,000||
+||Organize the most engaged hackathon in Africa courtesy Celo.|Logistics & Welfare for Educators: $1,500||
+||Welcome developers of all level to build using Celo smart contract toolkit|Videography & Photography: $1,500||
+||Organize 3 - 7 days hackathon and incubation program|Merch & Celo Swag packs - $1,500: For attendees, participants, judges and workers.||
+||Guide these projects to testnet, MVP and mainnet respectively.|General logistics - $1,000||
+||Invite successful project founders to mentor participants in this program.|Backdrops & banner - $700||
+||Program will be hybrid|||
+||Onchain certificate of participation will be issued to all participants|||
+|**Content creation & distribution**|Web 3 education, blockchain development and other content will be created.|Ask: $5,500|6 months|
+||Weekly update about Celo will be highlighted|Video creation||
+||Bi-weekly articles about web 3 in relation to Celo ecosystem|Bi-weekly articles||
+||Video contents|Sponsored ad||
+||Animation explanation of crypto terms in relation to Celo|Uploads and distribution on social platforms||
+|**Blockchain clubs & communities**|Establish sub communities across cities and institutions in Nigeria & Africa|**Ask: $2,400 for 5 schools - $12,000**|6 months|
+||Onboard students projects to Celo blockchain|Venue: $250||
+||Host seminar and meet-up in 5 school and cities.|Welfare for educators & speakers: $350||
+||Bounties for active web 3 participants in colleges and cities|Transportation & accommodation: $1,000||
+|||Bounties for school ambassadors and active participants: $300||
+|||Refreshment: $500||
+|**Start-up pitch exhibition**|Support 5 - 10 start-ups look to integrate Celo to their product.|**Ask - $5,000**: For 3-5 brilliant start-up utilizing Celo.|3-6 months|
+||Onboard 2- 5 businesses to Celo|||
+||Onboard project from other chains to Celo.|||
+||Support Climate neutral initiatives utilizing Celo blockchain.|||
+|**Operational Cost - Less than 10% of budget**|4 Core team members will work for 20 hours per week|**Ask: $5,500**|6 months|
+
+Social Media:
+
+* [Dapps over Apps on X (twitter) ](https://www.twitter.com/dappsoverapps.com/)
+* [Telegram ](https://t.me/+OrxSivWwoh1jZDU0)
+* [Dappsoverapps@gmail.com](mailto:Dappsoverapps@gmail.com)
+* [Instagram](https://www.instagram.com/yestodapps)
+
+Multisig:
```

Comment:
Please add the multisig address(`0x79d4881D34B6159aBcAb35Ad5029a4DE66c28626`) which is in the payload for visibility.

Reply:
Hi @0xzoz Thank you for your time and patience. Corrections have been made accordingly. Thanks once again!